otelcol/internal/grpclog/logger.go

```diff
@@ -27,7 +27,7 @@ func SetLogger(baseLogger *zap.Logger, loglevel zapcore.Level) *zapgrpc.Logger {
 			c = core
 		}
 		return c.With([]zapcore.Field{zap.Bool("grpc_log", true)})
-	})))
+	}), zap.AddCallerSkip(5)))
```

Comment:
you can probably add a test for this. There is already a unit test that validates some other aspects, you can add a check that the caller is `logger_test.go`

Reply:
Okay i will work on it.